homeassistant/components/epson/media_player.py

```diff
@@ -53,6 +53,7 @@ async def async_setup_entry(
 ) -> None:
     """Set up the Epson projector from a config entry."""
     projector: Projector = hass.data[DOMAIN][config_entry.entry_id]
+    assert config_entry.unique_id
     projector_entity = EpsonProjectorMediaPlayer(
         projector=projector,
         name=config_entry.title,
```

Comment:
This is only used for the device, and thus can be committed, as de device name will automatically fallback to the entry title.